viadot/sources/business_core.py

```diff
@@ -128,29 +128,37 @@ def get_data(self) -> Dict:
         logger.info("Data was downloaded successfully.")
         return json.loads(response.text)
 
-    def to_df(self, if_empty: str = "skip") -> pd.DataFrame:
+    def to_df(self, if_empty: Literal["warn", "fail", "skip"] = "skip") -> pd.DataFrame:
```

Comment:
The function converts `dict` to DF. Is the dict required as input or is it handled by the `self` variable?

Reply:
It's handled by `self`.